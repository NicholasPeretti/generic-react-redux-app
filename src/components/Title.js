import React from 'react'

const style = {
  fontSize: '32px',
  fontWeight: 'bold'
}

const Title = ({ children }) => (
  <h1 style={style}>
    {children}
  </h1>
)

export default Title
