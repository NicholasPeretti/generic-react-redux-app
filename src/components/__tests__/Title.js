import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import Title from '../Title'

describe('Title', () => {
  PresentationalTest(Title)

  test('Should match snapshot', () => {
    const wrapper = shallow(<Title>This is a title</Title>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
