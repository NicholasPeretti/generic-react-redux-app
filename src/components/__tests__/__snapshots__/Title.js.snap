// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Should match snapshot 1`] = `"<h1 style=\\"font-size:32px;font-weight:bold\\">This is a title</h1>"`;
